flake.nix

```
@@ -36,35 +36,24 @@
 
         toolkit = self.lib.basicTools pkgs;
 
-        callPackage = pkgs.newScope (
-          pkgs
-          // { inherit callPackage; }
-          // packages
-        );
-        callPy3Package = pkgs.newScope (
-          pkgs
-          // pkgs.python3Packages
-          // { inherit callPackage; }
-          // packages
-        );
+        loom = (self.lib.mkScopes pkgs).callPy3Package ./pkgs/loom { };
 
-        packages = {
+        ociImgLoom = pkgs.callPackage ./pkgs/oci/inferenceql.loom {
```

```suggestion
        ociImgLoom = scopes.callPackage ./pkgs/oci/inferenceql.loom {
```

Then you can just include `loom` in the arguments of `oci/inferenceql.loom/default.nix`